File: src_simple/f90/calc_val_fonction_1.f90
! calc_val_fonction_1  -  11.03.2014

! ******************************************
! calcule des valeurs de la fonction f(x) = x^2 pour x dans [-5 ; 5]
! ******************************************

program calc_val_fonction_1
    
    implicit none

    ! declaration
    integer :: x

    ! boucle sur les valeurs de x
    do x = -5, 5, 1
        write(*, *) x, x**2
    end do

end program calc_val_fonction_1

File: src_simple/python/calc_val_fonction_1.py
#!/usr/bin/env python
# -*- coding=utf-8 -*-

""" calc_val_fonction_1  -  11.03.2014 """

def calc_val_fonction_1():
    """ calcule des valeurs de la fonction f(x) = x^2 pour x dans [-5 ; 5] """

    for x in range(-5, 6, 1):
        print(x**2)

if __name__ == "__main__":
    calc_val_fonction_1()

